plugins/lookup/mergelist.py

```diff
@@ -0,0 +1,75 @@
+# (c) 2020, Jose Angel Munoz <josea.munoz(at)gmail.com>
+# (c) 2020 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    lookup: mergelist
+    author: Jose Angel Munoz <josea.munoz@gmail.com>
+    short_description: merges lists from a given array key
+    description:
+      - This lookup returns the contents of two or more merged lists using a common key.
+    options:
```

Comment:
You need to document the option `key`.

Reply:
Added. Silly me, I didn't realize carefully about your comment. Thank you!